lib/utmp.c

```diff
@@ -417,11 +416,8 @@ active_sessions_count(const char *name, unsigned long limit)
 			continue;
 		}
 		count++;
-		if (count > limit) {
-			break;
-		}
```

Comment:
Why is this moved to the while condition?  Is it necessary, or is it cosmetic?  If the latter, please do it in a separate commit.